templates/test/result.html.ep

```diff
@@ -55,10 +57,21 @@
                 </li>
             </ul>
 
-            <div class="tab-content">
-                <div role="tabpanel" class="tab-pane active" id="details">
-                    %= include 'test/details'
-                </div>
+            <div class="tab-content" style="height: 100%">
+                % if ($show_details) {
```

Comment:
this also includes the change to not show the "details" tab at all in case there are no modules but also no modules.

Reply:
But hopefully this doesn't influence running tests. These will obviously no details from until the test order is parsed and uploaded. (The 'Details' tab is updated via JavaScript in this case.)

Reply:
actually I can leave out that part for another PR, updated.